notify_python35.cpp

```diff
@@ -297,33 +296,22 @@ bool NotifyPython35::notify(const std::string& deliveryName,
 			    const string& triggerReason,
 			    const string& customMessage)
 {
+	//lock_guard<mutex> guard(m_configMutex);
```

Comment:
We should probably hold this mutex here. We don't want this function to interleave with (re)configure. E.g. in reconfigure, we do "m_pFunc = NULL" and here we are using "m_pFunc".